mlpipeline/pipeline/component_abc.py

```diff
@@ -0,0 +1,41 @@
+import abc
+
+
+class ComponentABC(abc.ABC):
```

Comment:
I think this abstraction isn't necessary and could be merger with `Component`.

Reply:
just separation of `execute` implementation, if someone had an idea of having a different behaviour